src/main/java/com/google/api/codegen/viewmodel/OptionalArrayMethodView.java

```diff
@@ -71,6 +73,14 @@ public static Builder newBuilder() {
     return new AutoValue_OptionalArrayMethodView.Builder();
   }
 
+  public boolean isReturnEmpty() {
+    return !hasReturnValue();
+  }
+
+  public boolean hasRequestStreaming() {
```

Comment:
That doesn't actually work - if it has request streaming, there will be multiple requests.